example/variables.tf

```diff
@@ -3,8 +3,18 @@
  * the next 3 variables are automatically supplied by the pipeline.
  *
 */
-variable "cluster_name" {
+variable "eks_cluster_name" {
+  description = "The name of the cluster (eg.: cloud-platform-live-0)"
```

Comment:
Can we change this to `The name of the eks cluster to retrieve the OIDC information` or something like that.